src/views/SpaceProposals.vue

```diff
@@ -166,7 +166,9 @@ watch(
             :voted="userVotedProposalIds.includes(proposal.id)"
             :hide-space-avatar="proposal.space.id === space.id"
             :to="{
-              path: `${domain ? '' : proposal.space.id}/proposal/${proposal.id}`
+              path: domain
+                ? `proposal/${proposal.id}`
+                : `/${proposal.space.id}/proposal/${proposal.id}`
```

Comment:
Can't you use normal route with param here? If we change the route path someday this will break.

Reply:
Should work like this 
```
:to="{
              name: 'spaceProposal',
              params: { id: proposal.id, key: proposal.space.id }
            }"```